packages/common/src/operations.graphql

```diff
@@ -18,6 +27,7 @@ query invoice($id: String!) {
             itemCode
             itemId
             itemName
+            itemUnit
```

Comment:
this adds some data which we weren't actually querying for into the detail query